Description: Remove escape key from filebrowser
 Some shells send escape-key-sequences for the arrow keys. So the filebrowser
 should not be closed when using arrow keys.
Author: Johann Felix Soden <johfel@debian.org>
Bug-Debian: http://bugs.debian.org/444394

--- a/plugin/filebrowser.vim
+++ b/plugin/filebrowser.vim
@@ -147,7 +147,6 @@
 function! <SID>FB_SetMaps()
 	nnoremap <buffer> <silent> q :bdelete<cr>
 	nnoremap <buffer> <silent> C :call FB_DisplayFiles(getcwd())<CR>
-	nnoremap <buffer> <silent> <esc> :bdelete<cr>
 	nnoremap <buffer> <silent> <CR> :call <SID>FB_EditEntry()<CR>
 	nnoremap <buffer> <silent> ? :call <SID>FB_ToggleHelp()<CR>
 
@@ -197,7 +196,7 @@
 		let txt = 
 			\  "\" <cr>: on file, choose the file and quit\n"
 			\ ."\"       on dir, enter directory\n"
-			\ ."\" q/<esc>: quit without choosing\n"
+			\ ."\" q: quit without choosing\n"
 			\ ."\" C: change directory to getcwd()\n"
 			\ ."\" ?: toggle help verbosity\n"
 			\ ."\"= ".getcwd()
